lib/dai-plugin-mcd/src/index.js

```diff
@@ -71,29 +76,39 @@ const defaultCdpTypes = [
 ];
 
 export default {
-  addConfig: (_, { cdpTypes = defaultCdpTypes, addressOverrides } = {}) => {
+  addConfig: (
+    _,
+    { cdpTypes = defaultCdpTypes, network = 'mainnet', addressOverrides } = {}
+  ) => {
     if (addressOverrides) {
       addContracts = mapValues(addContracts, (contractDetails, name) => ({
         ...contractDetails,
-        address: addressOverrides[name] || contractDetails.address
+        address: addressOverrides[name] || contractDetails.address[network]
       }));
     }
-
     const tokens = uniqBy(cdpTypes, 'currency').map(
       ({ currency, address, abi }) => {
         const data =
           address && abi ? { address, abi } : addContracts[currency.symbol];
         assert(data, `No address and ABI found for "${currency.symbol}"`);
-        return { currency, ...data };
+        return { currency, abi: data.abi, address: data.address[network] };
       }
     );
-
+    //remove contracts that don't have an address for the given network
```

Comment:
which ones were you finding didn't have addresses here, btw? i think the set should match... it might be good to remove this actually in order to surface issues with configuration earlier

Reply:
never mind this -- i'm about to add something to the testnet addresses that won't have a kovan equivalent :)

Reply:
Looks like they do now match now with the current snapshot, but I left this code there as you suggested.